community-membership.md

```diff
@@ -181,6 +181,7 @@ The following apply to the subproject for which one would be an owner.
 - Make and approve technical design decisions for the subproject.
 - Set technical direction and priorities for the subproject.
 - Define milestones and releases.
+  - Decides on when PRs are merged to control the release scope. 
```

Comment:
is there indentation issue?

Reply:
I made it a sub-item of a release control.